### PR TITLE
Scalar Operations

### PR DESCRIPTION
**Pull request by stefandraht**
I've added scalar versions of multiply and divide along with their component variations. I did not add scalar versions of add or subtract as I don't think it technically makes sense to add a vector by a scalar. As @JoonasSalmela suggested, I have divide by zero return a zero vector (like in threejs) or zero component.

### CODE REVIEW

File: build/victor.js [hunk 1]
```diff
@@ -419,6 +491,64 @@ Victor.prototype.multiply = function (vector) {
 };
 
 /**
+ * Multiplies both vector axis by the given scalar value
+ *
+ * ### Examples:
+ *     var vec = new Victor(100, 50);
+ *
+ *     vec.multiply(2);
```
Comment (JoonasSalmela) [inline build/victor.js]: Should be `vec.scalarMultiply(2);`.

File: build/victor.js [hunk 2]
```diff
@@ -419,6 +491,64 @@ Victor.prototype.multiply = function (vector) {
...
+	return this;
+};
+
+/**
+ * Multiplies the X axis by the given scalar
+ *
+ * ### Examples:
+ *     var vec = new Victor(100, 50);
+ *
+ *     vec.multiplyX(2);
```
Comment (JoonasSalmela) [inline build/victor.js]: Scalar here too.

File: build/victor.js [hunk 3]
```diff
@@ -419,6 +491,64 @@ Victor.prototype.multiply = function (vector) {
...
+ *     vec.toString();
+ *     // => x:100, y:100
```
Comment (JoonasSalmela) [inline build/victor.js]: Y is multiplied here instead of X.

File: build/victor.js [hunk 4]
```diff
@@ -419,6 +491,64 @@ Victor.prototype.multiply = function (vector) {
...
+	return this;
+};
+
+/**
+ * Multiplies the Y axis by the given scalar
+ *
+ * ### Examples:
+ *     var vec = new Victor(100, 50);
+ *
+ *     vec.multiplyX(2);
```
Comment (JoonasSalmela) [inline build/victor.js]: `vec.scalarMultiplyY(2);`

**Comment by JoonasSalmela** [comment]
The documentation has some errors. Commented on some of them.

**Comment by stefandraht** [comment]
Thanks!

**Comment by neojski** [comment]
@maxkueng, @stefandraht, any chance of merging this?

**Comment by maxkueng** [comment]
I've changed the method names to `divideScalar()` instead of `scalarDivide()` and `divideScalarX()` instead of `scalarDivideX()`. It's more in line with the rest. Like, we have `rotateDeg()` instead of `degRotate()`.

I've also merged `addScalar()` by @JoonasSalmela and added the X/Y-only versions and the same for subtract.

**Pull request merged**